### ISSUE TITLE
user should be allowed to define global defaults

### ISSUE THREAD

**Issue by ssbarnea** [impredicative/irc-rss-feed-bot#32]
Instead of using YAML anchors which are not very easy to understand (or popular) and which do add their own boilerplate, it would be very useful to make defaults configurable.

Example:
```
defaults:
      period: 0.5
      shorten: false
      new: 0
```
Adding something like this would allow users to avoid having to either repeat themselves or use the yaml anchors.

**Comment by impredicative** [comment]
This is not a resolution yet, but I just want to say that editing YAML is so much easier with a good YAML capable editor that lets you navigate easily within the file, and also does basic syntax error checking. FWIW, PyCharm supports it well.

**Comment by impredicative** [comment]
@ssbarnea This has now been implemented and released as the tagged version 0.5.0. It supports the `defaults` keys `shorten` and `new`. It has currently not been implemented for `period` because it's doubtful that anyone would want a default period > 1.

Be aware that setting a default of `shorten: false` does not excuse from providing the `BITLY_TOKENS` environment variable. This environment variable would still be required. I would think of it as requiring a credit card when using the free tier of a cloud provider.

**Comment by impredicative** [comment]
I will go ahead and close this issue since I believe it has been resolved. I look forward to your comment if any.
